src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesIfStatement.java

```diff
@@ -17,7 +17,7 @@
 
     void method(String sectionName) {
         if ("Content".equals(sectionName) || "Overview".equals(sectionName)
-                || (!"AbbreviationAsWordInName".equals(sectionName) // violation
+                || (!"AbbreviationAsWordInName".equals(sectionName) // violation '.* expression'
```

Comment:
Same here, `.* expression` is not very descriptive.

Reply:
Done

Reply:
Resolved